conf/systemd.service

```diff
@@ -6,12 +6,15 @@ After=network.target
 Type=simple
 User=__APP__
 Group=__APP__
-WorkingDirectory=__FINALPATH__
+WorkingDirectory=__FINALPATH__/
 Environment="PATH=__ENV_PATH__"
 Environment="THELOUNGE_HOME=/home/yunohost.app/__APP__/"
 Environment="NODE_ENV=production"
 ExecStart=/usr/bin/yarn start
-Restart=always
+Restart=on-failure
+RestartSec=5
+StartLimitInterval=60s
+StartLimitBurst=3
 
 [Install]
-WantedBy=default.target
```

Comment:
@alexAubin The problem was this line. (It took me so long to figure this out...🙄)